core/src/repair/ancestor_hashes_service.rs

```diff
@@ -784,26 +784,16 @@ impl AncestorHashesService {
     ) {
         dead_slot_pool.retain(|dead_slot| {
             let epoch = root_bank.get_epoch_and_slot_index(*dead_slot).0;
-            if let Some(epoch_stakes) = root_bank.epoch_stakes(epoch) {
+            //TODO: figure out if we even need to make this check
+            if let Some(_epoch_stakes) = root_bank.epoch_stakes(epoch) {
```

Comment:
We no longer *need* to fetch the epoch_stakes from root_bank (as they are available from cluster_slots) but if this check has some other function maybe we still want it?